backend/app.js

```diff
@@ -1,16 +1,18 @@
 const express = require('express');
-const app = express();
+var mongoose = require('mongoose');
```

Comment:
Would a `const` here work?

Reply:
Yeah, there's a bunch of instances throughout the codebase for that. I'll open up an issue for it.

Reply:
On second thought, I'll fix it here. We can probably just bring it up in code reviews.